vendor/Makefile

```diff
@@ -0,0 +1,152 @@
+VENDOR_DIR = $(shell pwd)
+BUILD_DIR = $(VENDOR_DIR)/build
+
+THRIFT_VERSION = 0.9.2
+CODEMIRROR_VERSION = 5.25.0
+JSPLUMB_VERSION = 2.2.0
+MARKED_VERSION = v0.3.3
+JQUERY_VERSION = 3.1.1
+DOJO_VERSION = 1.10.4
+HIGHLIGHT_VERSION = 9.0.0
+
+default: build
+
+dep: $(VENDOR_DIR)/thrift
+dep: $(VENDOR_DIR)/codemirror
+dep: $(VENDOR_DIR)/codemirror/dist/codemirror.min.js
+dep: $(VENDOR_DIR)/jsplumb
+dep: $(VENDOR_DIR)/marked
+dep: $(VENDOR_DIR)/jquery/jquery-$(JQUERY_VERSION)-min.js
+dep: $(VENDOR_DIR)/dojotoolkit
+dep: $(VENDOR_DIR)/highlightjs/highlight.min.js
+dep: $(VENDOR_DIR)/highlightjs/css/xcode.min.css
+
+vendor_dir:
+	mkdir -p $(VENDOR_DIR)
+
+$(VENDOR_DIR)/thrift: vendor_dir
```

Comment:
Please add a comment that we use only the thrift js file, that is why the download needed.